src/Utilities/UnitTests/project.json

```diff
@@ -1,6 +1,7 @@
 ﻿{
   "dependencies": {
     "xunit": "2.1.0",
+    "xunit.netcore.extensions": "1.0.0-prerelease-00116",
```

Comment:
I heard a rumor that in the future, you might actually be able to add comments *directly in the project.json file*. :astonished: